<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandomSelectGUIClass</class>
 <widget class="QWidget" name="RandomSelectGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>338</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>点名机</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QLabel" name="Welcome">
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>欢迎使用！</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="SelectFile">
        <property name="text">
         <string>名单文件：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="FilePath">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="SelectButton">
           <property name="text">
            <string>选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Reload">
           <property name="text">
            <string>重新加载文件内容</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="NormalRandomMode">
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="text">
           <string>普通模式</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="EnhanceRandomMode">
          <property name="text">
           <string>增强随机模式</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="Start">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>开始抽签</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Stop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>停止</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="Result">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>结果：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Name">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RandomSelectGUI.qrc"/>
 </resources>
 <connections/>
</ui>
